nightfall-deployer/src/contract-setup.mjs

```diff
@@ -15,7 +15,7 @@ async function setupCircuits() {
   // when deploying on infura
   // do serial registration to predict nonce
   // or, if we have the owner's private key, sign with that, rather than use an unlocked account
-  if (config.USE_INFURA || config.ETH_PRIVATE_KEY) {
+  if (config.USE_INFURA || config.ETH_PRIVATE_KEY || config.USE_HOSTED_GETH) {
```

Comment:
`USE_HOSTED_GETH` is not set or used anywhere

Reply:
Removed